docs/community/com-config/com-config-sql-cli-enable.md

```diff
@@ -36,3 +36,4 @@ These flags can be used within the CLI.
 ## Further information
 
 * [FeatureBase SQL guide](/docs/sql-guide/sql-guide-home)
+* [SQL CLI HTTPS examples](/docs/community/com-api/old-sql-endpoint)
```

Comment:
Not a CLI. It's an endpoint

Reply:
Ok so the anchor text should be "SQL HTTPS endpoint examples"